primitives/chain-westend/src/lib.rs

```diff
@@ -57,6 +57,9 @@ pub const FROM_WESTEND_LATEST_CONFIRMED_NONCE_METHOD: &str = "FromWestendInbound
 /// Name of the `FromWestendInboundLaneApi::unrewarded_relayers_state` runtime method.
 pub const FROM_WESTEND_UNREWARDED_RELAYERS_STATE: &str = "FromWestendInboundLaneApi_unrewarded_relayers_state";
 
+/// The length of a session (how often authorities change) on Westend measured in of number of blocks.
+pub const SESSION_LENGTH: BlockNumber = 10 * time_units::MINUTES;
```

Comment:
Note that afair, this doesn't necessarily need to be accurate - the session may finish earlier, so it's rather a max/desired session length.

Reply:
Not saying that I don't believe you, but can you point to any docs/code where this is the case?

Reply:
Session is controlled via [`should_session_end`](https://github.com/paritytech/substrate/blob/3472a656d23a8bb0850a074b13deac2a91b2e6e3/frame/session/src/lib.rs#L139) and it can implement arbitrary logic really.

If you look at [BABE implementation](https://github.com/paritytech/substrate/blob/3472a656d23a8bb0850a074b13deac2a91b2e6e3/frame/babe/src/lib.rs#L475), there are some edge cases in case of missed slots, where the session ends a bit ealier than compared to uniform periodic session.